src/index.html

```diff
@@ -130,7 +130,7 @@ <h3 class="text--headline text--headline--3 space--mt--large space--mb">The Don
           <li><strong>Do not</strong> participate for the gold medal in the “Oppression Olympics” (you don’t need to compare how your struggle is just as bad)</li>
           <li><strong>Do not</strong> behave as though you know best</li>
           <li><strong>Do not</strong> take credit for the labor of those who are marginalized and did the work before you stepped into the picture</li>
-          <li><strong>Do not</strong> assume that every member of an underinvested group feels oppressed</li>
```

Comment:
Hi @jagawag! Thanks for this. The use of "underinvested" was a conscious decision. People from "marginalized" groups are neither marginalized nor underrepresented. We exist, we are here. The issue is that we are ignored, and as such, underinvested: in time, resources, energy, and money.